### ISSUE TITLE
Incorrect parsing of timestamp's milliseconds with a leading zero

### ISSUE THREAD

**Issue by akarmes** [koxudaxi/local-data-api#90]
**Describe the bug**

Given a column of type timestamp, leading zero in milliseconds is ignored. Example: inserted value "2021-03-03T08:31:15.077Z" will be read as "2021-03-03 08:31:15.770000"

**To Reproduce**

Given following table:

```sql
SELECT column_name, data_type FROM information_schema.columns WHERE table_name = 'solutions';

# id | character varying
# created_at | timestamp without time zone
```

And following test script

```js
const AWS = require('aws-sdk');

const resourceArn = '';
const secretArn = '';
const database = '';
const endpoint = '';

const rdsDataService = new AWS.RDSDataService({ endpoint });

const executeSql = (sql) => rdsDataService.executeStatement({ resourceArn, secretArn, database, sql }).promise();

const test = async (sql) => {
  await executeSql(`INSERT INTO solutions (id, created_at) VALUES ('1', '2021-03-03T08:31:15.077Z')`);
  const { records } = await executeSql(`SELECT * FROM solutions WHERE id = '1'`);
  console.log(records);
}
```

The output would be following:

```js
[
  [
    { stringValue: '1' },
    { stringValue: '2021-03-03 08:31:15.770000' },
  ]
]

```

**Expected behavior**

Is should behave equally to AWS Aurora Data API and return:

```js
[
  [
    { stringValue: '1' },
    { stringValue: '2021-03-03 08:31:15.077' },
  ]
]
```

**Comment by koxudaxi** [comment]
@akarmes 
Thank you for creating this issue.
OK, I will fix it!!

**Comment by akarmes** [comment]
@koxudaxi Thank you for your fix! I've tried `0.5.6` and now output format is correct in the sense that it returns 3 digits for milliseconds. 

But the other issue still persists: it has a bug with leading zero. The value `2021-03-03 00:00:00.055` will be parsed as `2021-03-03 00:00:00.550`.

**Comment by koxudaxi** [comment]
@akarmes 
hmm, I can't reproduce the promble.
```sh
$ aws rds-data execute-statement --database 'test' \
    --resource-arn $RDS_DATA_API_CLIENT_RESOURCE_ARN \
    --secret-arn $RDS_DATA_API_CLIENT_SECRETARN \
    --include-result-metadata \
    --sql $'insert into example(t) values (\'2020-03-11 11:04:21.055\')'
```
```
 "stringValue": "2020-03-11 11:04:21.055"
```

Also, I used this logic to get the first three numbers.
I don't understand why the first zero is dropped. 🤔 
https://github.com/koxudaxi/local-data-api/blob/dde8601489c102464e9f79f929f828e3be50988b/local_data_api/resources/resource.py#L335-L337

**Comment by koxudaxi** [comment]
I'm sorry, you are correct.
I ran the command to real Data-API 😢 

I found the bug in the driver.
https://github.com/baztian/jaydebeapi/blob/cd2fd4c6c047803916fc92a534f02f1213e252d6/jaydebeapi/__init__.py#L619-L625
```python
d = d.replace(microsecond=int(str(java_val.getNanos())[:6]))
```
https://github.com/baztian/jaydebeapi/pull/177
This PR has not merged yet. 😓

**Comment by koxudaxi** [comment]
@akarmes 
I have released a new version as `0.5.7`
It passed the integration test.
https://github.com/koxudaxi/local-data-api/blob/18a8d3fd0f025eeb8acac09e075e25b99e75be59/scripts/integration-test.sh#L103-L104

**Comment by akarmes** [comment]
Thank you @koxudaxi! I can confirm the issue is resolved.

**Comment by akarmes** [comment]
@koxudaxi I've tested version 0.5.7 more extensively and noticed, that given a row with `null` value for column of a timestamp type, local-data-api is returning string "None". I'll post an example later

**Comment by koxudaxi** [comment]
@akarmes 
I just fixed it and released `0.5.8` 🚀 
Thank you very much.

The wrapper of the JDBC driver for Python is buggy.
I want to replace this implementation with Kotlin when I get the time 😅 
https://github.com/koxudaxi/local-data-api/issues/70

**Comment by akarmes** [comment]
I was just writing next bug report 😅 Thank you again!
